ansible_deployer/command_line.py

```diff
@@ -502,13 +502,13 @@ def main():
         sys.exit(2)
     options = parse_options(sys.argv[1:])
 
-    log_dir = None
```

Comment:
maybe we'll add a configuration or option for log_dir one day to allow file logging for other commands?